source/com.microsoft.tfs.util/src/com/microsoft/tfs/util/temp/TempStorageService.java

```diff
@@ -343,8 +344,19 @@ public synchronized void forgetItem(final File item) {
     public synchronized void renameItem(final File sourceItem, final File targetItem) {
         Check.notNull(sourceItem, "sourceItem"); //$NON-NLS-1$
         Check.notNull(targetItem, "targetItem"); //$NON-NLS-1$
-        Check.isTrue(sourceItem.exists(), "renaming: source file does not exist"); //$NON-NLS-1$
-        Check.isTrue(!targetItem.exists(), "renaming: target file already exists"); //$NON-NLS-1$
+
+        if (!sourceItem.exists()) {
+            throw new RuntimeException(
+                MessageFormat.format(
+                    Messages.getString("TempStorageService.RenameErrorSourceDoesNotExistFormat"), //$NON-NLS-1$
+                    sourceItem.getAbsolutePath()));
+        }
+        if (targetItem.exists()) {
+            throw new RuntimeException(
+                MessageFormat.format(
+                    Messages.getString("TempStorageService.RenameErrorTargetExistsFormat"), //$NON-NLS-1$
```

Comment:
Same as above, message string has {0} and {1} params but only one source item is passed.